crypto/x509/x509_req.c

```diff
@@ -227,13 +227,21 @@ X509_ATTRIBUTE *X509_REQ_get_attr(const X509_REQ *req, int loc)
 
 X509_ATTRIBUTE *X509_REQ_delete_attr(X509_REQ *req, int loc)
 {
-    return X509at_delete_attr(req->req_info.attributes, loc);
+    X509_ATTRIBUTE *attr = NULL;
+
+    attr = X509at_delete_attr(req->req_info.attributes, loc);
+    if (attr)
```

Comment:
Please change further (for simplification and adherence to the [coding style guidelines](https://github.com/openssl/technical-policies/blob/103a28c62e8cc9d46865203368d20c358c6c4e9a/policies/coding-style.md#chapter-15-expressions)) to:
```
 X509_ATTRIBUTE *attr = X509at_delete_attr(req->req_info.attributes, loc);
 
 if (attr != NULL)
 ```